database/database.js

```diff
@@ -0,0 +1,124 @@
+const mongoose = require('mongoose');
+const fs = require('fs');
+const path = require('path');
+const promise = require('bluebird');
+const rawPath = path.join(__dirname + '/../restaurant.txt');
+let readFile = promise.promisify(fs.readFile);
+
+
+mongoose.connect('mongodb://student:student1@ds149742.mlab.com:49742/restaurants');
+
+let restaurantSchema = {   "id": {type:Number, unique: true},
```

Comment:
Not sure if its githubs display UI again, but possible that indentation is off for the schema?

Reply:
spacial issues addressed with ESLint :+1: